src/man/sssd-ldap.5.xml

```diff
@@ -938,7 +938,7 @@
                             </itemizedlist>
                         </para>
                         <para>
-                            Default: false
```

Comment:
Hi,

I'd like to suggest to add a `Recommened for security reasons` to the `true` item explanation, similar to the note at the `false` item. And with this I would add an explanation that `allow` is used as a default for compatibility reasons but we recommend to check that TLS is really used e.g. by checking the logs for related warnings.

bye,
Sumit

Reply:
Fixed, thank you.

Reply:
Thanks, what about adding an explanation why `allow` is used as default? If you think that it is not needed for this in-between version I'm fine.

Reply:
ah, it's in a different patch, but I agree with Alexey it better fits in the patch where the default is changed.